lightgbm/lightgbm_terminator.py

```diff
@@ -0,0 +1,55 @@
+"""
```

Comment:
Please add the execution setting of `lightgbm_terminator.py` to the CI.

LightGBM tests are configured in this file
https://github.com/optuna/optuna-examples/blob/main/.github/workflows/lightgbm.yml#L37-L38